templates/test/running_table.html.ep

```diff
@@ -6,11 +6,11 @@
 	    { targets: 0,
               className: "name" },
             { targets: 3,
+              className: "time",
               "render": function ( data, type, row ) {
-                  console.debug(new Date(data));
-		  if (type === 'display')
-		      return jQuery.timeago(new Date(data + " UTC"));
-		  else
+                    if (type === 'display') {
+		      return jQuery.timeago(new Date(data));
+                  } else
 		      return data;
```

Comment:
I don't understand your question to be honest - so let me explain the context:

this is the render function of the time column - it's task is to transform what's in the <td> to HTML that can be displayed. It can render it differently for displaying than for querying thought.

So if you search in the table for Mon, you actually get the test results for monday even though the display is "4 days ago". That's what the type argument is for - and we only do the timeago widget for displayed entries, otherwise we leave the pure data.

The timeago function actually creates a widget that will update as time goes on. It will be 10 minutes when you load and 11 minutes ago if you wait. You don't want to do that for !displayed entries. You could do it, but it's wasteful